diff --git a/libtd/src/v7a/java/org/drinkless/tdlib/package-info.java b/libtd/src/v7a/java/org/drinkless/tdlib/package-info.java
new file mode 100644
index 0000000..2c78d3c
--- /dev/null
+++ b/libtd/src/v7a/java/org/drinkless/tdlib/package-info.java
@@ -0,0 +1,4 @@
+@javax.annotation.ParametersAreNonnullByDefault
+package org.drinkless.tdlib;
+
+import javax.annotation.ParametersAreNonnullByDefault;
